K8s-deployment/Charts/auditing-server/example-secrets/secrets/config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "version": "1.0",
+  "modules": [
+    {
+      "id": "iudx.auditing.server.rabbitmq.RabbitMQVerticle",
+      "verticleInstances": 1,
+      "dataBrokerIP": "",
```

Comment:
please fill in the details wherever possible, like the rmq ip/domain, port to connect to using k8s services . See : https://github.com/datakaveri/iudx-deployment/blob/master/K8s-deployment/Charts/catalogue/example-secrets/secrets/config.json for example

Reply:
use ``<>`` placeholders with description on what to fill, wherever default details can't be filled